.vscode/settings.json

```diff
@@ -2,5 +2,29 @@
     "cSpell.words": [
         "ASPNETCORE",
         "Cuit"
-    ]
+    ],
+    "rest-client.environmentVariables": {
+        "$shared": {
+            "apisIntHost": "apisint.fromdoppler.net",
+            "apisQaHost": "apisqa.fromdoppler.net",
+            "apisProdHost": "apis.fromdoppler.com",
```

Comment:
I known that is just a name but right now we are using `APIS_DOMAIN` to reference this urls in Swarm stacks ([example](https://github.com/MakingSense/doppler-swarm/blob/master/custom-domain-stack/int.env#L2))

*I can not made a code suggestion on this PR, but I am thinking in something like that:
            `"apisIntDomain": "apisint.fromdoppler.net",`
            `"apisQaDomain": "apisqa.fromdoppler.net",`
            `"apisProdDomain": "apis.fromdoppler.com",`

Reply:
Sounds good, thanks!

Reply:
Done in 6fc5db55f54f1b0cee05ee382cb62eb5353cd773

(I will rebase before merging)